rsearch/Godeps/Godeps.json

```diff
@@ -0,0 +1,23 @@
+{
+	"ImportPath": "github.com/romana/contrib/rsearch",
+	"GoVersion": "go1.5",
+	"GodepVersion": "v58",
+	"Deps": [
```

Comment:
It is not explicitly mentioned - i guess this is something we will tackle when receive first feedback from k8s.

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md#godep-and-dependency-management